sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidget.kt

```diff
@@ -12,7 +12,7 @@ import com.mapbox.maps.MapboxExperimental
 @MapboxExperimental
 open class BitmapWidget @JvmOverloads constructor(
   bitmap: Bitmap,
-  position: WidgetPosition = WidgetPosition {
+  private val position: WidgetPosition = WidgetPosition {
```

Comment:
Nit: even though it's private,  maybe rename it to `originalPosition` to avoid any confusion with `getPosition()`.

Reply:
sounds good, do you think we should expose it as public API?

Reply:
No, I don't think we need to expose it for now.
Let's do so when there's a valid use-case. Moreover, the user can keep a copy of it around.